core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1926,13 +1926,27 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
 
     for {
       googleProjectId <- traceWithParent("getGoogleProjectFromBuffer", span)(_ => resourceBufferService.getGoogleProjectFromBuffer(projectPoolType, workspaceId))
+      _ <- traceWithParent("maybeMoveGoogleProjectToFolder", span)(_ => maybeMoveGoogleProjectToFolder(billingProject.servicePerimeter, googleProjectId))
```

Comment:
with PPW, we decided to give Terra SAs/groups (Rawls SA, Cromwell SA, billing@fc.org, dev group for non-prod envs) permissions at the folder level instead of the google project level. Are all of the service perimeter folders set up in the same way right now or will we need to add those?

Reply:
https://broadworkbench.atlassian.net/browse/CA-1314

Reply:
permissions are inherited from the environment level folder

Reply:
Ok great. Does that mean we should wait on merging this until CA-1314 is done? I think if we need CA-1314 in order to have all of the environment-level folder permissions for these service perimeter folders/projects.